src/state/homepage/homepage.reducer.ts

```diff
@@ -0,0 +1,28 @@
+import { Survey } from '../../common/types/survey';
+
+export interface HomepageState {
+    surveys: Survey[];
+}
+
+export const initialHomepageState = {
+    surveys: [
```

Comment:
Just adding some placeholder data here, not sure if it's the right place.

Reply:
It works for now, ideally we want to pass state to the `createStore` function (we can do this conditionally to populate only in dev mode), see: https://redux.js.org/recipes/structuring-reducers/initializing-state

Reply:
Please review my [latest commit](https://github.com/decrn/localsurvey/pull/23/commits/0c6162d25bcab4e9641a05bcebb0fecd28ce9d03)